applications/crossbar/test/cb_cdrs_test.erl

```diff
@@ -0,0 +1,44 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2010-2018, Voxter Communications Inc
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Ben Bradford
+%%%-------------------------------------------------------------------
+-module(cb_cdrs_test).
+-include_lib("eunit/include/eunit.hrl").
+
+%% Test handle_utc_time_offset function with different server
+%% time zones and offsets
+handle_utc_time_offset_test_() ->
+    %% Simulate a time stamp from a CDR
+    %% (generated in freeswitch depending on system TZ)
+    LocalTimestamp = 63687252084,
+
+    %% Depending on the systems timezone, Calculate the UTC time from the LocalTimestamp
+    LocalDateTime = calendar:gregorian_seconds_to_datetime(LocalTimestamp),
+    UTCDateTimeList = calendar:local_time_to_universal_time_dst(LocalDateTime),
+    UTCTimestamp = calendar:datetime_to_gregorian_seconds(lists:last(UTCDateTimeList)),
```

Comment:
Also, this unit test is not useful at all. It is literally copy/paste of the implementation. Instead, I would like to see something like:

```erlang
Tests = [{63687252084, 63687237684, -14400}],
[?_assertEqual(Local, handle_utc_time_offset(UTC, Offset)) || {UTC, Local, Offset} <- Tests].
```